daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala

```diff
@@ -50,6 +50,31 @@ class EncodeValueSpec extends AnyFreeSpec with Matchers {
         |    c = 42
         |  c = M.R3""".stripMargin.replace("\r\n", "\n")
     }
+    "tuple" in {
+      def tupleId(n: Int): v.Identifier = v
+        .Identifier()
+        .withPackageId("40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7")
```

Comment:
can we get the package id from daml-lf/language?

Reply:
We could, but the lookup key is `daml-prim-DA-Types-40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7`. So, we'd still end up hard-coding the hash.

Reply:
ah fair enough that’s not very helpful